tests/digital_identity_service/index.spec.js

```diff
@@ -26,7 +26,7 @@ describe('DigitalIdentityService', () => {
   describe('#createShareSession', () => {
     const setupResponse = (responseBody, responseStatusCode = 200) => {
       nock(apiUrlDomain)
-        .post(new RegExp('/v2/sessions'))
+        .post(new RegExp('/v2/sessions[^/]'))
```

Comment:
Why was that needed ?

Reply:
Because this regex also matches the URL for `createShareQrCode` which is `/v2/sessions/${sessionId}/qr-codes`. This was causing nock to give the wrong response. Even when I tried to do `nock.cleanAll()` after the tests.

Reply:
Ok, I see!